### ISSUE TITLE
[QST] Integer Data Types are available for Conv2d fprop?

### ISSUE THREAD

**Issue by IzanCatalan** [NVIDIA/cutlass#1979]
**What is your question?**
Hi, I would like to know if I can perform a conv2d fprop with int8 data type, in c++ kernels and in python.

Thanks.

**Comment by jackkosaian** [comment]
Yes, it is possible. For example, if you configure CUTLASS with `cmake .. -DCUTLASS_NVCC_ARCHS="80"`, you can see an INT8 Conv2d  kernel emitted under `tools/library/generated/conv2d/80/i16832fprop_optimized_s8/cutlass_tensorop_i16832fprop_optimized_s8_256x128_64x3_nhwc_align16.cu`

**Comment by IzanCatalan** [comment]
@jackkosaian are they available in Python? I couldn't check that as I have written in a [new post](https://github.com/NVIDIA/cutlass/issues/1990)
